src/engine/enginevumeter.cpp

```diff
@@ -93,6 +99,24 @@ void EngineVuMeter::process(CSAMPLE* pIn, const int iBufferSize) {
     } else {
         --m_peakDuration;
     }
+
+    if (clipped & 1) {
```

Comment:
I think I would prefer to calculate the old peak now as 
"m_ctrlPeakIndicatorR || m_ctrlPeakIndicatorL" than there is no doubt that all indicators are synced.